src/parsers/perf/perfparser.cpp

```diff
@@ -812,6 +812,9 @@ class PerfParserPrivate : public QObject
     {
         Data::BottomUp::initializeParents(&bottomUpResult.root);
 
+        for (auto it = tracepointCostIdLookup.begin(); it != tracepointCostIdLookup.end(); it++) {
```

Comment:
can you add a comment why this is needed? why do we want to have the total application time as total cost? is this just for tooltips to compute a sane relative cost?

Reply:
I wasn't sure what to use as total cost. I thought that if hotspot tells you, that function x spends 30% of its runtime in syscall y, its quite useful.

Reply:
But the other way around (30% of the runtime in syscall x was spent in function y) is also valid.
Maybe I should add a settings to choose ether application runtime or total syscall runtime as totalCost.

Reply:
ok, then keep it as-is and add a comment to indicate that this is to get costs relative to the total runtime in tooltips etc.